src/Asset.js

```diff
@@ -174,9 +174,11 @@ class Asset {
 
     // If this asset is main file of the package, use the sanitized package name
     if (this.name === main) {
-      const packageName = sanitizeFilename(this.package.name, {
-        replacement: '-'
-      });
+      const packageName =
+        this.options.outFile ||
```

Comment:
This currently only gets used when the asset matches `main` in package.json. I guess I'd expect it to also work when it doesn't match, or there is no `main` defined (below case). Maybe add an additional case above this if statement to always return `this.options.outfile` if it is defined?